api/src/Entity/User.php

```diff
@@ -268,6 +270,23 @@ public function isEmployee(): bool
         return false;
     }
 
+    public function isAssociatedWithRepairer(int|string|Repairer $repairer): bool
```

Comment:
Is there a case where `$repairer` could be a string?

Reply:
> Is there a case where `$repairer` could be a string?

I should have asked instead: ensure that `$repairer` could be only Repairer or (int or string but not both):
`uriTemplate: '/repairers/{repairer_id}/appointments'`
Is `repairer_id` a string or int ?